src/object/cli_obj.c

```diff
@@ -1421,7 +1421,7 @@ dc_obj_open(tse_task_t *task)
 	if (rc)
 		D_GOTO(fail_layout_created, rc);
 
-	obj->cob_ec_parity_rotate = 0;
```

Comment:
Why not define "ec_parity_rotate", then set "obj->cob_ec_parity_rotate = ec_parity_rotate" as test logic does. That will be more convenient for enable/disable the EC rotation.

Reply:
oh, this is temporary solution, I will remove all of these parity_rotate flags, once I added the layout version.